RELEASING.md

```diff
@@ -65,14 +68,32 @@ Create tags after a release that match the version number, `major.minor.patch`,
 
 For a discussion on whether to add a prefixing `v` to release tags, see [#35](https://github.com/opensearch-project/.github/issues/35).  
 
-## Release Labels
+## Releasing
+
+### Release Labels
 
 Repositories create consistent release labels, such as `v1.0.0`, `v1.1.0` and `v2.0.0`, as well as `patch` and `backport`. Use release labels to target an issue or a PR for a given release. See [MAINTAINERS](MAINTAINERS.md#triage-open-issues) for more information on triaging issues.
 
-## Releasing
+## Release Scheduling
+
+Because OpenSearch currently releases as a bundle, the OpenSearch release process is centralized and has several steps along the way.  As the first step in this release process, the Engineering Effectiveness team creates a release ticket in the [opensearch-build repo](https://github.com/opensearch-project/opensearch-build) and assigns it to an overall release manager (example: [2.0 release ticket](https://github.com/opensearch-project/opensearch-build/issues/1650)).  The team also creates individual release issues per repo with what's expected to go in that release. If you have a question about a release, you can always comment on a release issue using @opensearch-project/engineering-effectiveness.  
+
+### Release Schedule Date Definitions
+
+#### Feature Freeze
+All code changes that add new capabilities to OpenSearch need to be merged to "main" and backported to the planned release. Additional merges/backports can happen for bugs fixes and CVEs.
+
+#### Code Freeze
+This is date where we expect everything is working and there are no known issues.  Freezing the code before the release date gives us a chance to perform long-running performance tests and do other final verifications.  No more changes can be merged to the release branch after this point except for fixes to blocking bugs.  This generally happens 1 week before the release date for minor versions.
```

Comment:
As described 'freeze' is a suggestion, as changes will automatically be accepted from repositories and built automatically via the build system.   I suggest that we add a release process on that date to update the manifest.yml to static commit ids to make code freeze a concrete milestone.  By creating an opportunity to approve/reject a change to the manifest ensures the change is properly triaged and reviewed.

The mechanics of this process need not be described fully here as they can be added in the release issue.  This seems like a way of getting a better handle on the 'bounce' of bugs before a release is ready.

How does this phrasing occur to you? 

> No more changes can be merged to the release branch after this point except for fixes to blocking bugs.

_Changes in repository branches no longer be added to the distribution build by default, to update component(s) a pull request must be created with a justification for why the change is needed, after this has been approved and merged the distribution build will update._

Reply:
I like that, but I think let's say it has to be approved by a repo admin.  WDYT?

Reply:
By manifest.yml, I mean the version specific one in opensearch-build repository, [opensearch-2.0.0.yml](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/2.0.0/opensearch-2.0.0.yml), there is only one to manage.  Consider this scenario which has happened many times in past releases

1) Security plugin has a bug fix. 
2) Security team deems its worth doing and merging into their 2,1,0 branch.
3) [NEW] Then they cut a request to update the 2.10 manifest to pull in the fix
4) The head honcho reviews the request.
5) [NEW] They decides it doesn't meet the requirements for the 2.1.0 release, close the pull request unmerged.

This speed bump to ensure consistent standard are used and allows for staging/management of multiple plugins at the same time.

Reply:
I think it's the "release manager". There's always a next release on all lines possible. We should assign such a person every time we increment a version and a ticket is cut.

Reply:
You know, the more I think about this, the more I think we're once again pulling on the thread of governance.  I think I'd like to put this in as is, and we can follow up with additional PRs as we think more about governance structure.

Reply:
I think there could still be a cleaner transition into release milestone

> After code freeze, a release candidate is proposed.  Long running release validation activities are started, such as performance and stress testing.  To ensure these results are representative of the final release only blocking issues are accepted.
>
>Release candidate results from validation activities are gathered, reviewed and reported against the release criteria, if they are outside expectations another release candidate is created and this process is restarted.  After the release candidate passes the quality criteria, the promotion to final release beings.

Reply:
I've updated per the first change set.

We didn't do multiple RC for 1.0, and they're not planned for 2.0.  I think we should noodle more on what should the release of 3.0 look like, specifically  what are those exit criteria.   Put let's get on the other side of 2.0 and then revisit this.